main.rb

```diff
@@ -1,3 +1,165 @@
 # rubocop:disable Style/CaseEquality
+# rubocop:disable Metrics/ModuleLength
+module Enumerable
+  def make_array(input)
+    return input.to_a if input.class.name == 'Range'
 
+    input
+  end
+
+  def my_each
+    if block_given?
+      i = 0
+      while i < length
```

Comment:
Another option could be `times`: `size.times do |i|`.

Reply:
Nice trick next I will try to use it more thank you